test/unit/app/controllers/permissions/permissions-controller-test.js

```diff
@@ -823,9 +823,9 @@ describe('permissions controller', function () {
 
     beforeEach(function () {
       identities = ALL_ACCOUNTS.reduce(
-        (identities, account) => {
-          identities[account] = {}
-          return identities
+        (acc, account) => {
```

Comment:
IMO `identities` is more descriptive here, maybe `identitiesAcc` would be better? Maybe I'm being picky about variable names too... feel free to challenge this :D